### DISCUSSION TITLE
Does veramo support offline credential exchange?

### DISCUSSION THREAD

**Discussion by thomas-tran** [decentralized-identity/veramo#407] [Q&A]
For example, the university sends the certificate credentials to graduate students. However, some students may not have the mobile agent open or turn off their mobile. How can the student accept the credential when they are online?

**Comment by jasheal** [comment]
Hi, 

Veramo is not opinionated about the transport layer. It would depend on the transport protocol you choose for your app. If an app is offline there is no way to send any communication except some situations where the user scans a QR code or maybe via bluetooth.
